Docs/features/task-search.md

```diff
@@ -0,0 +1,23 @@
+#### Task Search
+
+As of 0.4.12, Singularity has better support for searching historical tasks. A global task search endpoint was added:
+
+`/api/history/tasks` -> Retrieve the history sorted by startedAt for all inactive tasks.
+
+... and the above endpoint as well `/api/history/request/{requestId}/tasks` now take additonal query parameters:
```

Comment:
This line is kind of confusing to me. Maybe 'The above endpoint as well as `/api/history/request/{requestId}/tasks` now take additonal query parameters'